### PR TITLE
Change the assets/vendors folder to assets/static

### LINKED ISSUE #10: Rename `src/assets/vendors` to `src/assets/static`
Followup once https://github.com/Shopify/slate/issues/411 has been closed and released.

### PR DESCRIPTION
**Pull request by t-kelly**
Fixes #10 

BREAKING CHANGE - Wait until Slate v1.0.0-beta.1 has been released before merging.

### CODE REVIEW

File: slate.config.js [hunk 1]
```diff
@@ -8,8 +8,13 @@ const alias = {
 };
 
 module.exports = {
-  extends: {
-    dev: {resolve: {alias}},
-    prod: {resolve: {alias}},
+  slateCssVarLoader: {
+    cssVarLoaderLiquidPath: [path.resolve(__dirname,'src/snippets/css-variables.liquid')],
```
Comment (chrisberthe) [inline slate.config.js]: Do we need to include `path.resolve`?

Comment (tauthomas01) [reply slate.config.js]: If I understood correctly, it generates the absolute path so this line will do : 

`/Users/myUsername/slate/src/snippets/css-variables.liquid`?

Comment (t-kelly) [reply slate.config.js]: Tested it without `path.resolve()` and without `__dirname`. Worked 👍. Will remove.

Comment (t-kelly) [reply slate.config.js]: Now that I think of it -- this should be in a seperate PR. Moving it.

**Pull request merged**